tests/loggers/test_wandb.py

```diff
@@ -16,11 +16,11 @@ def test_wandb_logger(wandb):
     logger = WandbLogger(anonymous=True, offline=True)
 
     logger.log_metrics({'acc': 1.0})
-    wandb.init().log.assert_called_once_with({'acc': 1.0})
+    wandb.init().log.assert_called_once_with({'acc': 1.0}, step=None)
 
     wandb.init().log.reset_mock()
-    logger.log_metrics({'acc': 1.0}, step=3)
-    wandb.init().log.assert_called_once_with({'global_step': 3, 'acc': 1.0})
+    logger.log_metrics({'acc': 1.0}, step=None)
+    wandb.init().log.assert_called_once_with({'acc': 1.0}, step=None)
```

Comment:
```suggestion
    logger.log_metrics({'acc': 1.0}, step=3)
    wandb.init().log.assert_called_once_with({'acc': 1.0}, step=3)
```
I think we should keep the test for int, because the None case is already tested above.